lib/ReactViews/SelectableDimensions/Select.tsx

```diff
@@ -88,3 +92,52 @@ export const SelectableDimensionEnum: React.FC<{
     />
   );
 });
+
+export const SelectableDimensionEnumMulti: React.FC<{
```

Comment:
Could provide some jsdoc to show distinction between SelectableDimensionEnumMulti and SelectableDimensionEnum

Reply:
done!